app/serializers/api/v2/quotas/quota_closed_and_transferred_event_serializer.rb

```diff
@@ -12,7 +12,20 @@ class QuotaClosedAndTransferredEventSerializer
           event.transferred_amount.try(:to_f)
         end
 
-        has_one :quota_definition, serializer: Api::V2::QuotaOrderNumbers::QuotaDefinitionSerializer
```

Comment:
We have issues with recursing over measures and then definitions and then this event and back again in the parser on the frontend if we try and be good boys with jsonapi

Reply:
I'd need to lazily load related resources from an accumulating graph of type/id includes with attributes if I wanted to fix this in the frontend which is quite a significant change